backend/src/middleware/auth.js

```diff
@@ -34,11 +35,16 @@ const auth = (req, res, next) => {
 
         // check whether key is a lead's key or a member's key
         // this is used by the leadsOnly middleware
-        if (key.endsWith(leadSuffix)) {
+        if (key.endsWith(leadSuffix) || key.endsWith(directorSuffix)) {
           req._is_lead = true
         } else {
           req._is_lead = false
         }
+        if (key.endsWith(directorSuffix)) {
+          req._is_director = true

```

Comment:
line 38's change is smart but I think removing the `|| endsWith(directorSuffx)` from there and doing this might be clearer even if it's a couple more lines of code.
```suggestion
if (key.endsWith(directorSuffix)) {
          req._is_director = true
          req._is_lead = true
} else {
           req._is_director = true
          req._is_lead = true
}
```